openedx/core/djangoapps/credit/tests/test_models.py

```diff
@@ -85,3 +85,57 @@ def add_credit_course(self):
         credit_course = CreditCourse(course_key=self.course_key, enabled=True)
         credit_course.save()
         return credit_course
+
+
+class CreditRequirementStatusTests(TestCase):
+    """
+    Tests for credit requirement status models.
+    """
+
+    def setUp(self):
+        super(CreditRequirementStatusTests, self).setUp()
+        self.course_key = CourseKey.from_string("edX/DemoX/Demo_Course")
+
+    def test_retire_user(self):
```

Comment:
It would be good to also have a test that ensures `False` is returned when you retire a user who has no CreditRequirementStatuses (you could do a lot of the setup below in your `setUp` method).